tests/unit/test_exceptions.py

```diff
@@ -14,35 +14,37 @@
 
 import http.client
 import json
+from google.rpc import status_pb2
+from google.rpc.status_pb2 import Status
 
 import grpc
 import mock
 import requests
-
+from google.rpc import error_details_pb2
+from google.rpc import code_pb2
+from google.protobuf import any_pb2, json_format
+from grpc_status import rpc_status
```

Comment:
Likewise, these imports should be conditional.

Reply:
The only conditional import should be `grpc_status` right? Assuming `error_details_pb2` and `any_pb2` come from `googleapis-common-protos`; `google.protobuf` is a common dependency.

Reply:
Strictly, the `grpc` import should also be conditional.  Also, all the non-stdlib and non-`google.api_core` imports should be grouped together and sorted.